src/overlays/actors/ovl_Bg_Relay_Objects/z_bg_relay_objects.c

```diff
@@ -10,17 +10,28 @@
 
 #define THIS ((BgRelayObjects*)thisx)
 
+typedef enum {
+    WINDMILL_ROTATING_GEAR,
+    WINDMILL_DAMPE_STONE_DOOR
+} WindmillSetpiecesMode;
+
 void BgRelayObjects_Init(Actor* thisx, GlobalContext* globalCtx);
 void BgRelayObjects_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgRelayObjects_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgRelayObjects_Draw(Actor* thisx, GlobalContext* globalCtx);
 
+void func_808A90F4(BgRelayObjects* this, GlobalContext* globalCtx);
+void func_808A91AC(BgRelayObjects* this, GlobalContext* globalCtx);
+void func_808A9234(BgRelayObjects* this, GlobalContext* globalCtx);
+void BgRelayObjects_Noop(BgRelayObjects* this, GlobalContext* globalCtx);
+void func_808A932C(BgRelayObjects* this, GlobalContext* globalCtx);
+void func_808A939C(BgRelayObjects* this, GlobalContext* globalCtx);
+
 extern UNK_TYPE D_060001A0;
 extern UNK_TYPE D_060003C4;
 extern UNK_TYPE D_06001AB0;
 extern UNK_TYPE D_060025FC;
```

Comment:
Based on their usage, the display lists can be typed correctly here:
```suggestion
extern Gfx D_060001A0[];
extern UNK_TYPE D_060003C4;
extern Gfx D_06001AB0[];
extern UNK_TYPE D_060025FC;
```